src/interfaces/store-interfaces/state-interfaces/restoration-state-interface.ts

```diff
@@ -0,0 +1,29 @@
+import { CorpTypeCd, RestorationTypes } from '@/enums'
+
+export interface RestorationNameRequestIF {
+  legalName: string
+  legalType: CorpTypeCd
+  nrNumber: string
+}
+
+export interface RestorationStateIF {
+  businessInfoValid: boolean
+  businessNameValid: boolean
+  type: RestorationTypes
+  expiry?: string // YYYY-MM-DD
+  // defineBusinessValid: boolean
+  // startDate: string
+  // businessAddress: BusinessAddressIF
+  // businessNumber?: string
+}
+
+export const EmptyRestorationState: RestorationStateIF = {
+  businessInfoValid: false,
+  businessNameValid: false,
+  type: null,
+  expiry: null
+  // defineBusinessValid: false,
+  // startDate: '',
+  // businessAddress: null,
+  // businessNumber: null
+}
```

Comment:
^^ I thought I'd follow my own advice and I created this "empty" object. I use it while initializing the store (state model).